config/jobs/kubernetes-csi/external-snapshotter/external-snapshotter-config.yaml

```diff
@@ -36,6 +36,8 @@ presubmits:
           value: "1.17"
         - name: CSI_PROW_DRIVER_VERSION
           value: "v1.4.0"
+        - name: CSI_SNAPSHOTTER_VERSION
```

Comment:
does this mean that changes to the snapshot-controller in external-snapshotter prs are untested?

Reply:
It is tested: https://storage.googleapis.com/kubernetes-jenkins/pr-logs/pull/kubernetes-csi_external-snapshotter/425/pull-kubernetes-csi-external-snapshotter-1-17-on-kubernetes-1-17/1326879789335711744/artifacts/default/csi-hostpath-snapshotter-0/csi-snapshotter.log

Reply:
https://storage.googleapis.com/kubernetes-jenkins/pr-logs/pull/kubernetes-csi_external-snapshotter/425/pull-kubernetes-csi-external-snapshotter-1-17-on-kubernetes-1-17/1326879789335711744/artifacts/default/snapshot-controller-0/snapshot-controller.log

Reply:
CSI_SNAPSHOTTER_VERSION is set in prow.sh currently.
https://github.com/kubernetes-csi/csi-release-tools/blob/master/prow.sh#L340

Reply:
What I mean is that it looks like prow.sh installs a version of the snapshot controller that has already merged and been released? https://github.com/kubernetes-csi/csi-release-tools/blob/master/prow.sh#L722

So if you make changes to snapshot-controller in a PR, then those changes won't get tested as part of the pull job.

Reply:
Looks like that's the case.  We probably need to replace the snapshot-controller image with the one built from the PR.

Reply:
For images used by the csi-driver-host-path deploy script, overriding the images that were built as part of the repo under test is defined here:
https://github.com/kubernetes-csi/csi-release-tools/blob/5d874cce4e649dfd254d01b9b44179ffa72aee75/prow.sh#L1047-L1079

Something similar is needed for the snapshot controller and prow.sh itself.

But this is unrelated to this PR, right?

Reply:
Yes, I think so.  That will be a separate PR to fix prow.sh.